activestate.yaml

```diff
@@ -137,41 +137,20 @@ scripts:
     if: eq .OS.Name "Windows"
     value: |
       set -e
-      if [ "$#" -ne 4 ]; then
-        echo "Required Arguments: <Owner/Name> <Version> <Release Notes URL> <Shortcuts> (comma-separated list of shortcuts to build)"
-        exit 1
-      fi
-
-      export GOFLAGS='-mod=vendor'
-      dir=$(cmd //C echo %CD%)
-      cp installers/msi-language/Product.p.wxs installers/msi-language/Product.wxs
-
-      pad () {
-        string=$1
 
-        # Use base length that avoids string overflow warnings from wix
-        # Subtract string length and brackets
-        padLength=$((246-${#string}-4))
-        printf -v pad '%*s' "$padLength"
-        padded=$(printf '%s' $1)$(echo ${pad// /-})
-        echo $padded
-      }
+      PROJECT_OWNER_NAME=${2:-ActiveState/ActivePerl-5.26}
```

Comment:
Why do we have a default value for this argument? If the only argument passed was `Perl` wouldn't this script still fail when we try to run the Go script?

Reply:
This is to support the no-argument option that you also pointed out below.   You are right I can remove that.